lib/worker-fn.js

```diff
@@ -0,0 +1,41 @@
+'use strict'
+
+const { realImport, realRequire } = require('real-require')
+
+async function getWorkerFunction (t) {
```

Comment:
I'd like to see a jsdoc block that describes the purpose of this function, its parameters, and what it returns/throws.